### PR TITLE
Fix bbox not being set on routing result

### LINKED ISSUE #36: problem in fit bounds for identical points
http://gh-maps-react.s3-website.eu-central-1.amazonaws.com/?point=14.239884671170358%2C51.38756272791164&point=14.239884671170358%2C51.38756272791164&vehicle=car

### PR DESCRIPTION
**Pull request by Janekdererste**
fixes #36

### CODE REVIEW

**Review by karussell — APPROVED** [review]
Thanks!

File: src/App.tsx [hunk 1]
```diff
@@ -32,15 +32,15 @@ export default function App() {
...
     // only use the api info's bbox until any other bounding box was chosen. Is this too messy?
-    const chooseBoundingBox = function (infoBbox: Bbox, pathBbox: Bbox, shouldUseInfoBbox: boolean) {
-        if (shouldUseInfoBbox && pathBbox.every(num => num !== 0)) {
+    const chooseBoundingBox = function (infoBbox: Bbox, shouldUseInfoBbox: boolean, pathBbox?: Bbox) {
+        if (shouldUseInfoBbox && pathBbox && pathBbox.every(num => num !== 0)) {
             setUseInfoBbox(false)
             return pathBbox
         } else if (shouldUseInfoBbox) return infoBbox
-        return pathBbox
+        return pathBbox || [0, 0, 0, 0]
```
Comment (karussell) [inline src/App.tsx:40]: btw: we use this BBox as the "invalid case" here and elsewhere, but strictly speaking this is a perfect valid bbox :)

Still should not make a problem as this bbox is unrealistically small and points to nowhere

Comment (Janekdererste) [reply src/App.tsx:40]: The problem here was that the path coming from the api doesn't have a bbox property for routes with the same start and end point. Mapbox can actually handle the bbox with no area. 

The rest of the code relies on the bbox being at least present.

Comment (karussell) [reply src/App.tsx:40]: > The problem here was that the path coming from the api doesn't have a bbox property for routes with the same start and end point. 

Do you mean we include the bbox in the response of /route and do not include it when two coordinates are identical? Or do you mean with "api" our JS code?

**Comment by karussell** [comment]
The problem is now that it zooms very close to the route even if it is very small and once it is the same points it zooms out to world wide. Instead we should probably define a minimum bbox **and** additionally ensure that the markers are inside this bbox.

File: src/App.tsx (hunk 1 above)
Comment (karussell) [reply src/App.tsx:40]: Oh, indeed. This is a bug I think.

Comment (karussell) [reply src/App.tsx:40]: https://github.com/graphhopper/graphhopper/issues/2277

**Comment by karussell** [comment]
https://github.com/graphhopper/graphhopper-maps/issues/53

**Pull request merged**